your-code/main.ipynb

```diff
@@ -207,6 +967,25 @@
     "### 10. Use a list comprehension to extract and print all values from the data set that are between 0.7 and 0.75."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[0.7347510852128797, 0.7025657075988269, 0.7132714209987683, 0.7280006345208285, 0.7258310816144985, 0.736029494090402, 0.7472962776291713, 0.7343092314001137, 0.7088042362916818, 0.7358941798147411, 0.7365416322783028, 0.7329024831531535, 0.7023481768180954, 0.7133695725698419, 0.7476323136755818, 0.7008930873050235, 0.7357618655811269, 0.7013619954419494, 0.7268545088531969, 0.7032248108904495, 0.7150799992177622, 0.7173964318670598, 0.7306642125209889, 0.7383982036983279, 0.7124169515124726, 0.71223975981198, 0.7298115760258881, 0.7051334608870933, 0.7321333453970806, 0.7178166705957928, 0.7374449181749492, 0.7159290265157912, 0.7294980584878026, 0.7215147068185102, 0.7045750112959754, 0.7111584953876777, 0.7209375615129192, 0.7396561066717461, 0.7303553716733908, 0.7133690799930356, 0.7390312927111312, 0.7356923324968598, 0.7078738356977186, 0.7321533684342264, 0.7490680818183477, 0.7130781095091867, 0.7085098152301276, 0.7264121114538001, 0.7196096371031628, 0.7393833032570581, 0.7244863338885654, 0.7033930409580709, 0.7274869207330491, 0.7376946446827494]\n"
+     ]
+    }
+   ],
+   "source": [
+    "lists = [df[col].tolist() for col in df.columns]\n",
+    "elements = [i for elem in lists for i in elem if i < 0.75 if i > 0.7]\n",
+    "print(elements)\n"
+   ]
```

Comment:
Funciona perfectamente pero se puede hacer en una linea. Y recuerda usar un **and** para unir ambos condicionales
```
[i[0] for col in df.columns for i in df[[col]].values if (i<0.75 and i>0.7)]
```
Otra historia es que pueda ser menos entendible :sweat_smile: